core/lib/lantern-trace-saver.js

```diff
@@ -117,19 +124,42 @@ function convertNodeTimingsToTrace(nodeTimings) {
   }
 
   /**
+   * @param {number} requestId
    * @param {LH.Artifacts.NetworkRequest} record
-   * @param {LH.Gatherer.Simulation.NodeTiming} timing
+   * @param {CompleteNodeTiming} timing
+   * @return {LH.TraceEvent}
+   */
+  function createWillSendRequestEvent(requestId, record, timing) {
```

Comment:
adding a `ResourceWillSendRequest` trace event for the navigation for completeness, though the perf panel doesn't seem to depend on it